### ISSUE TITLE
Final create page after the review page is just blank... and then the project never got uploaded :-(

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-2#301]
<img width="1055" alt="Screen Shot 2021-01-03 at 11 54 38 AM" src="https://user-images.githubusercontent.com/14945613/103484214-39f6de00-4ded-11eb-81a9-9dec20e9f49f.png">

Oh... now I notice the little tiny spinner in the corner.... it loops forever
<img width="57" alt="Screen Shot 2021-01-03 at 11 58 46 AM" src="https://user-images.githubusercontent.com/14945613/103484241-8510f100-4ded-11eb-8f7c-1031b47feed0.png">

**Comment by GriffGreen** [comment]
damn... no Commons Stack on Giveth 2 yet.... Sad Panda face

**Comment by mateodaza** [comment]
This is related to some changes on the impact graph, I guess it was bad day for testing it D:

Will let you know when it's alright

**Comment by GriffGreen** [comment]
No worries!

I can't do much for the dev meeting
so i figure i can help a little some how
:-D

**Comment by mateodaza** [comment]
This is a weird one, it worked for me. It's hard to follow up but will make sure of most edge cases, for example I will do my tests over Brave as much as Chrome

**Comment by jamespfarrell** [comment]
I can replicate:

mutation ($project: ProjectInput!) {
  addProject(project: $project) 
     id
     title
     description↵    admin↵    image↵    impactLocation↵    slug↵    walletAddress↵    categories {↵      name↵      __typename↵    }↵    __typename↵  }↵}↵
     
     variables: {project: {title: "Commons Stack",…}}
project: {title: "Commons Stack",…}
admin: "giveth"
categories: ["technology"]
description: "The Commons Stack isn't a platform, but rather a growing library of open-source components. Our designs and components are fully open-source and interoperable: you take what you need to incentivize work towards shared goals, provide accountability, facilitate governance, and receive feedback on the health of your Commons."
imageStatic: "2"
impactLocation: "Global"
title: "Commons Stack"
walletAddress: "0xE31B07525385Bc250dDbE4d7651Ea9991da04D9f"}
